packs/spell-effects/spell-effect-community-repair.json

```diff
@@ -4,7 +4,7 @@
     "name": "Spell Effect: Community Repair",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Community Repair]</p>\n<p>On a critical success, while a caster is within 60 feet of the site of the ritual, they receive a +1 status bonus to Will saves against emotion effects and a +10-foot status bonus to their Speeds. On a critical failure, while a caster is within 60 feet of the site of the ritual, they take a -10-foot status penalty to their Speeds as their movement slows to a morose crawl.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Community Repair]</p>\n<p>On a critical success, you receive a +1 status bonus to Will saves against emotion effects and a +10-foot status bonus to your Speeds.</p>\n<p>On a critical failure, you take a -10-foot status penalty to your Speeds.</p>"
```

Comment:
Should the `"community-repair-proximity"` predicate be `"terrain:community-repair"` instead?